Docs: Add a warning about non-recursive queries during domain bootstrap on autosecondary (#15043)
PowerDNS/pdns: docs/modes-of-operation.rst

Adding a autoprimary can be done either directly in the database,
or by using the 'pdnsutil add-autoprimary' command.

.. warning::

>   When secondary gets notification and is bootstraping a new domain using autosecondary feature it does
>   SOA and NS queries to IP address from ``supermasters`` table. These queries are NOT recursive!

Reviewer: ```suggestion
  When a secondary receives notification while bootstraping a new domain using autosecondary feature, it will send
  SOA and NS queries to the IP address obtained from the ``supermasters`` table. These queries are **not** recursive.
```

>   When a secondary receives notification while bootstraping a new domain using autosecondary feature, it will send
>   SOA and NS queries to the IP address obtained from the ``supermasters`` table. These queries are **not** recursive.

Reviewer 2: ```suggestion
  When a secondary receives notification while bootstrapping a new domain using autosecondary feature, it will send
  SOA and NS queries to the IP address matched in the ``supermasters`` table. These queries are **not** recursive.
```

>   It means that domain bootstrap will fail if your primary authoritative server is hidden behind recursor,
>   so make sure these queries goes (or routed by dnsdist) straight to auth server.

Reviewer: ```suggestion
  This will cause domain bootstrap to fail if the primary authoritative server is hidden behind a recursor,
  so make sure these queries go (or are forwarded by dnsdist) straight to the auth server.
```
